base-theme/layouts/partials/local_video_player.html

```diff
@@ -0,0 +1,28 @@
+{{ $random := delimit (shuffle (split (md5 "seed") "" )) "" }}
+<div class="video-container embedded-video-container">
+  <video
+    id="video-player-{{ .youtubeKey }}-{{ $random }}"
+    class="video-js vjs-default-skin vjs-big-play-centered vjs-16-9"
+    controls
+    data-setup='{
+    "fluid": true,
+    "inactivityTimeout": 0'
+  >
+      {{ if .archiveUrl }}
+        <source src="{{ partial "resource_url.html" (dict "context" .context "url" .archiveUrl) }}" type="video/mp4" />
+      {{ end }}
+      {{ if .captionsLocation }}
+        <track kind="captions" src="{{ .captionsLocation }}" mode="disabled" srclang="en" label="English">
+      {{ end }}
```

Comment:
As far as I can tell, offline captions (w/o youtube) don't work at all right now, either in the interactive transcript or in the player itself.

Maybe better to just leave these tracks out? That way there's no (non-functional) "CC" button. 

<img width="831" alt="Screenshot 2023-05-25 at 4 34 56 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/9010790/fa6d5a92-f163-4565-8cde-b6cc208ff823">